content/en/getting_started/agent/_index.md

```diff
@@ -12,90 +12,219 @@ further_reading:
-| **datadog.agent.running**        | Shows a value of `1` if the Agent is reporting to Datadog.                                                 |
-| **datadog.agent.started**        | A count sent with a value of `1` when the Agent starts (available in v6.12+).                                        |
+```text
+Datadog agent (v. 7.XX.X) started on <Hostname>
+```
 
-#### Checks
+#### Service Checks
 
-Depending on your platform, the Agent has several core checks enabled by default that collect metrics.
+The Agent is set up to provide the following service checks:
 
-| Check       | Metrics       | Platforms          |
-| ----------- | ------------- | ------------------ |
-| CPU         | [System][19]  | All                |
-| Disk        | [Disk][20]    | All                |
-| Docker      | [Docker][21]  | Docker             |
-| File Handle | [System][19]  | All except Mac     |
-| IO          | [System][19]  | All                |
-| Load        | [System][19]  | All except Windows |
-| Memory      | [System][19]  | All                |
-| Network     | [Network][22] | All                |
-| NTP         | [NTP][23]     | All                |
-| Uptime      | [System][19]  | All                |
-| Winproc     | [System][19]  | Windows            |
+  - **datadog.agent.up**:
+    Returns `OK` if the Agent is able to connect to Datadog.
+
+  - **datadog.agent.check_status**:
+    Returns `CRITICAL` if an Agent check is unable to send metrics to Datadog, otherwise returns `OK`.
+
+These checks can be used in the Datadog Platform to visualize the Agent status through Monitors and Dashboards at a quick glance. See [Service Check Overview][20] to learn more.
+
+#### Metrics
+
+In the Datadog UI, go to the Metrics Summary page **Metrics > Summary** and search for the metric **datadog.agent.started** or the metric **datadog.agent.running**. If these metrics are not visible right away, it may take a few minutes for the Agent to send the data to the Datadog Platform.
+
+Click on either of the metrics and a Metric panel will open to show additional metadata about where these Metrics are pulled from and any associated tags. Since no tags were configured on this host, you should only see the default tags that Datadog assigns to the metrics including `version` and `host`. See the following section on Agent Configuration Files to learn more about how to add tags.
+
+Explore other default metrics such as **ntp.offset** or **system.cpu.idle**
+
+## Agent Configuration Files
+
+The Agent's main configuration file is `datadog.yaml`. The required parameters are your [Datadog API key][15] which is used to associate your Agent's data with your organization and the Datadog site ({{< region-param key="dd_site" code="true" >}}). See the [sample config_template.yaml][21] for all available configuration options.
+
+Once you have the Agent installed, you can adjust the Agent configuration files to take advantage of other Datadog features including Tags
```

Comment:
```suggestion
You can adjust the Agent configuration files to take advantage of other Datadog features including tags.
```
Ah, here's the longer explanation i just requested :) Maybe a couple more examples of the features in the config?